test.ts

```diff
@@ -40,136 +72,718 @@ describe('does parsing', () => {
 
 describe('supported value types', () => {
   it('boolean without following argument', () => {
-
+    const [options] = parse(
+      ['-stars', '-q'],
+      {
+        'stars': {
+          alias: 's',
+          desc: 'oh I believe in yesterday.',
+          type: 'boolean'
+        },
+        'quiet': {
+          alias: 'q',
+          desc: 'the quiet one.',
+          type: 'boolean'
+        }
+      }
+    );
+    expect(options).toEqual({
+      stars: true,
+      quiet: true,
+    });
   });
 
   it("boolean with prefixed 'no'", () => {
-
+    const [options] = parse(
+      ['--no-stars', '--no-quiet'],
+      {
+        'no-stars': {
```

Comment:
--no-stars means value of 'stars' is false,
this test will never pass, please change the function call from `it` to `it.skip`.